api/admin/controller/__init__.py

```diff
@@ -259,7 +259,7 @@ def _check_redirect(target: str) -> Tuple[bool, str]:
         return True, ""
 
     @staticmethod
-    def redirect(target: str) -> Response:
+    def redirect(target: str) -> WerkzeugResponse:
```

Comment:
The response type here was a Flask Response and so is the return statement, should it be left as-is?

Reply:
I guess it works either way since it's the parent class

Reply:
The error from mypy if this is set to `Response`:
```
api/admin/controller/__init__.py:266: error: Incompatible return value type (got "werkzeug.wrappers.response.Response", expected "flask.wrappers.Response")
```